swan-lake/why-ballerina/flexibly-typed.md

```diff
@@ -292,7 +292,7 @@ $ bal run sample.bal
                                        </span>
                                     </td>
                                     <td>
-                                       <span class="cTableCode"><code>curl -d '{ "name": "John Little",  "birthYear": 1855 }' http://localhost:8080/record</code></span>
+                                       <span class="cTableCode"><code>curl -d '{ "name": "John Little", "birthYear": 1855 }' http://localhost:8080/record -H 'Content-Type: application/json'</code></span>
```

Comment:
Is this mandatory now?

Reply:
Yeah, data binding fails without it. 

https://github.com/ballerina-platform/ballerina-dev-website/pull/5689, https://github.com/ballerina-platform/ballerina-standard-library/issues/2530#issuecomment-1011271021